pkg/cluster/install.go

```diff
@@ -44,13 +43,37 @@ func (m *manager) AdminUpgrade(ctx context.Context) error {
 	return m.runSteps(ctx, steps)
 }
 
+func (m *manager) Update(ctx context.Context) error {
```

Comment:
follow-up: there probably needs to be a rebalancing of functions across cluster.go, install.go and some new files.